build.xml

```diff
@@ -1881,10 +1849,10 @@
       </java>
   </target>
 
-  <target name="_maybe_update_idea_to_java11plus" depends="init" unless="java.version.8">
-    <replace file="${eclipse.project.name}.iml" token="JDK_1_8" value="JDK_${ant.java.version}"/>
-    <replace file=".idea/misc.xml" token="JDK_1_8" value="JDK_${ant.java.version}"/>
-    <replace file=".idea/misc.xml" token="1.8" value="${ant.java.version}"/>
+  <target name="_maybe_update_idea_to_java11plus" depends="init">
```

Comment:
do we need this target? maybe we can update idea files directly?

Reply:
` maybe we can update idea files directly?`
What do you mean by directly? We need that token change for when we want to  build&run things on 17

Reply:
Clarified this one in a direct conversation. Jacek suggests adding any jvm options directly in the file.
Could be done in a follow up ticket
Also: they are not the same for both 11 and 17